models/issue_dependency.go

```diff
@@ -113,7 +113,10 @@ func issueDepExists(e Engine, issueID int64, depID int64) (bool, error) {
 
 // IssueNoDependenciesLeft checks if issue can be closed
 func IssueNoDependenciesLeft(issue *Issue) (bool, error) {
+	return issueNoDependenciesLeft(x, issue)
+}
 
+func issueNoDependenciesLeft(e Engine, issue *Issue) (bool, error) {
 	exists, err := x.
```

Comment:
Should this use e and not x ?
```suggestion
	exists, err := e.
```